src/inmanta/export.py

```diff
@@ -563,33 +565,33 @@ def export_dumpfiles(exporter: Exporter, types: ProxiedType) -> None:
         os.mkdir(prefix)
 
     for file in types["std::File"]:
-        path = os.path.join(prefix, file.host.name + file.path.replace("/", "+"))
+        path = os.path.join(prefix, file.host.name + file.path.replace("/", "+"))  # type: ignore
         with open(path, "w+", encoding="utf-8") as fd:
-            if isinstance(file.content, Unknown):
+            if isinstance(file.content, Unknown):  # type: ignore
                 fd.write("UNKNOWN -> error")
             else:
-                fd.write(file.content)
+                fd.write(file.content)  # type: ignore
 
     path = os.path.join(prefix, "services")
     with open(path, "w+", encoding="utf-8") as fd:
         for svc in types["std::Service"]:
-            fd.write("%s -> %s\n" % (svc.host.name, svc.name))
+            fd.write("%s -> %s\n" % (svc.host.name, svc.name))  # type: ignore
 
     path = os.path.join(prefix, "packages")
     with open(path, "w+", encoding="utf-8") as fd:
         for pkg in types["std::Package"]:
-            fd.write("%s -> %s\n" % (pkg.host.name, pkg.name))
```

Comment:
Why ignore this?

Reply:
Because the object is an inmanta entity. That is the plugin you are writing ;-p